tests/app/views/test_brief_response.py

```diff
@@ -367,26 +387,140 @@ def test_day_rate_should_be_less_than_service_max_price(self, live_dos_framework
         assert data == {'error': {'dayRate': 'max_less_than_min'}}
 
 
-class TestSubmitBriefResponse(BaseBriefResponseTest):
+class TestUpdateBriefResponse(BaseBriefResponseTest):
+    def _create_brief_response(self):
```

Comment:
This function is really similar to the one defined on the `BaseBriefResponseTest` class. Could they be reduced to one, passing in the different kwargs as needed?